Field-Robotics-Lab/dave
# Adding heavy weight torpedo model

Adds one more model - the Mark 48 heavy weight torpedo.

To test run
```
roslaunch dave_demo_launch dave_models.launch
```

You should see one additional model compared to https://github.com/Field-Robotics-Lab/dave/pull/209 - the large torpedo shown here:

![Screenshot from 2022-02-24 20-36-00](https://user-images.githubusercontent.com/8921143/155654592-14725afb-0adc-4210-b7d1-f6b5acc7f135.png)
